build.gradle

```diff
@@ -43,15 +43,6 @@ allprojects {
     apply plugin: 'com.netflix.nebula.info'
     apply plugin: 'org.jetbrains.kotlin.jvm'
 
-    // Set Kotlin api / language version to 1.7 since dgs-framework
```

Comment:
We are still stuck on gradle 7.x, so would still need this, no?

Reply:
We were able to move to Gradle 8.0.2, so this specific workaround is not needed.

Reply:
oh right, ok.